src/aind_data_transfer/config_loader/ephys_configuration_loader.py

```diff
@@ -1,18 +1,63 @@
 """Loads Ephys job configurations"""
 import argparse
+import json
+import logging
 import os
 import re
 from pathlib import Path
 
 import yaml
+from botocore.exceptions import ClientError
 from numcodecs import Blosc
 
 from aind_data_transfer.readers.ephys_readers import EphysReaders
+from aind_data_transfer.util.s3_utils import get_secret
 
 
 class EphysJobConfigurationLoader:
     """Class to handle loading ephys job configs"""
 
+    DEFAULT_AWS_REGION = "us-west-2"
+    # Not the actual passwords, just the key name to retrieve it from aws
+    # secrets manager
+    SERVICE_ENDPOINT_KEY = "service_endpoints"
+    VIDEO_ENCRYPTION_KEY_NAME = "video_encryption_password"
+    CODE_OCEAN_API_TOKEN_NAME = "codeocean-api-token"
+    # TODO: Is there a better way to do this?
+    CODE_REPO_LOCATION = (
```

Comment:
Probably in any of the YML files you're using to configure your uploaders?

Reply:
Based on how the library's being used, I'm hoping to sunset the yaml file since most of the defaults aren't changing much. I imagine some sort of introspection can be used? It's just trying to pull the package name for itself.